docs/resources/samples/clear-table-filter-for-active-cell.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Clear table column filter based on active cell location in Office Scripts'
+description: 'Learn how to clear table column filter based on active cell location.'
+ms.date: 03/02/2021
+localization_priority: Normal
+---
+
+# Clear table column filter based on active cell location
+
+Shows how to clear the table column filter based on active cell location. The script detects if the cell is part of a table and then determines the table column and clears any filter that may be applied on it.
+
+If you wish to learn more about how to save the filter prior to clearing it (and re-apply later), check out a [more advanced sample](move-rows-across-tables.md).
+
+## Video
+
+See [Range basics](range-basics.md#videos)
+
+## Screenshots
+
+_Before clearing column filter (notice the active cell)_
+
+![Before clearing column filter](../../images/before-filter-applied.png)
```

Comment:
This runs a bit counter to my earlier comment, but if we think the screenshots really help sell the description, maybe we should put them right after the "What this scripts does" text block.